masonite/exceptions.py

```diff
@@ -1,5 +1,13 @@
 class DriverNotFound(Exception):
     pass
 
+
 class DriverLibraryNotFound(Exception):
     pass
+
+
+class ExtensionException(Exception):
```

Comment:
Let's change this exception to `FileTypeException` instead.

Reply:
Ok!